src/flamegraph.js

```diff
@@ -63,12 +63,16 @@ export default function () {
     }
     var originalSearchHandler = searchHandler
 
-    let searchMatch = (d, term) => {
+    let searchMatch = (d, term, caseSensitive = false) => {
```

Comment:
Default to false to maintain current behavior.

Reply:
Made default behaviour back to case-sensitive. Pushed a new commit.